src/cff2pages/templates/md/base.md

```diff
@@ -1,7 +1,8 @@
 {%- from 'md/authors_macro.md' import authors_block -%}
 {%- from 'md/identifiers_macro.md' import identifier_block -%}
-{%- from 'md/reference_macro.md' import references_block -%}
+{%- from 'md/citation_macro.md' import cite_as -%}
 {%- from 'md/keyword_macro.md' import keyword_block -%}
+{%- from 'md/reference_macro.md' import references_block -%}
```

Comment:
Adjusting the order of imports to match the order in which macros are used in the template.